### ISSUE TITLE
Display the platform/user/etc. in every view of the GUI

### ISSUE THREAD

**Issue by kinow** [BSC-ES/autosubmit-gui#214]
At the moment I switch between tree view and quick view for testing suite experiments. The reason is that the quick view shows platform & user, useful information for the operator of the testing suite, however the tree view does not include that information.

e.g. quick view

![image](https://github.com/user-attachments/assets/c458b39f-72e7-4e54-a01c-db90f2b3caf3)

e.g. tree view

![image](https://github.com/user-attachments/assets/f70d456b-06e8-44dd-97a2-748a48846b11)

If that information were available in each view, then users would benefit from consistency (IMO) and would not have to switch between views when they needed to see the user of the experiment but were looking at the tree view.

**Comment by LuiggiTenorioK** [comment]
Agree, probably is better to have it in every view. I was also thinking of making an update of that upper bar to be collapsible so it can show more info if desired.

Something like this (expanded and possible sidebar change):

![image](https://github.com/user-attachments/assets/44f77624-b2aa-4cc3-996b-45ba67e0ec5d)


And (collapsed with a possible plot by status):

![image](https://github.com/user-attachments/assets/19bca824-bc6d-4391-8277-d0ad51c16f3c)

**Comment by kinow** [comment]
That works for me! Easier if I don't always have to click to expand the info tho (even if collapsed by default, if I expand then maybe save settings in local storage or something, so that next time I open that view it's already visible). Thanks!

**Comment by LuiggiTenorioK** [comment]
> That works for me! Easier if I don't always have to click to expand the info tho (even if collapsed by default, if I expand then maybe save settings in local storage or something, so that next time I open that view it's already visible). Thanks!

Yes, the idea is that it is collapsed by default and is always present in the experiment tabs. Not sure about keeping the settings in the local storage, some people might prefer to have it collapsed every time they enter an experiment but it will be kept as it is when changing tabs.

About the other two changes (sidebar style, donut chart by status with selection), I'll ask in one MWT meeting to see if users like that.

Also, this new upper bar solves #213!

**Comment by kinow** [comment]
> Also, this new upper bar solves https://github.com/BSC-ES/autosubmit-gui/issues/213!

Ah! That green circle will be used to indicate the status of the experiment? A circle with the status color? If so, hooray!
